bin/convert_dpc_folder.rb

```diff
@@ -47,7 +50,12 @@
   exit(false)
 end
 
-converter_args = [ options[:source], options[:target], options[:item_id_length], options[:checksums], options[:copy] ]
+converter_args = [ source: options[:source],
```

Comment:
I'm not sure why you're using this syntax here that creates a hash inside an array, since you splat the array later on. Just create a hash?